code/src/components/ErrorMessage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const backToStart = () => {
+    document.location.reload()
+  };
+
+const ErrorMessage = () => {
+    return (
+        <div className='error-message'>
+            <p>
+                Oops! Something went wrong. Your happy thought must be "lagom" as we say in Sweden. Between 5 and 140 characters
```

Comment:
Love the irony in your error message! "Your happy thought must be "lagom" as we say....